pipeline/Dockerfile

```diff
@@ -50,6 +54,9 @@ RUN apt-get update \
 
 WORKDIR /app
 ENV PATH="/app/bin:${PATH}"
+RUN echo "OCAML_BENCH_HOST=${OCAML_BENCH_HOST}}" > /app/.env
```

Comment:
```suggestion
RUN echo "OCAML_BENCH_HOST=${OCAML_BENCH_HOST}" > /app/.env
```

Reply:
Btw, does that work? The reason I did that for the frontend is because that's how `vite` loads env variables.

Shouldn't we do: `ENV OCAML_BENCH_HOST=${OCAML_BENCH_HOST}` instead?

Reply:
Just realised that we can load env variables like that for `vite` too. No need for creating an `.env` file

Reply:
Instead of declaring the env here, and passing args from docker-compose, we can just pass the environment variables from docker-compose. I think it's a cleaner approach, let me know what you think!

Reply:
Oh, I didn't know we could do that. That's definitely better indeed 🤗